.gitignore

```diff
@@ -2,4 +2,5 @@
 .infura
 .mnemonic
 node_modules/
+coverage.json
```

Comment:
What is this file and why is it ignored?

Reply:
Hi @SkandaBhat 
It's nothing but a coverage report.
It is generated every time, solidity-coverage is run.
Hence it's not needed in the repo.